tests/framework/Optimizers/fd_canyon.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.FD_Canyon</name>
+    <author>talbpaul</author>
+    <created>2018-09-05</created>
+    <classesTested>Optimizers.FiniteDifferenceGradientOptimizer</classesTested>
+    <description>
+      Originally this models tests minimizing $x y \cos(s+y)$, but has been extended to test the finite difference
+      gradient optimizer with denoising in 2 dimensions, to assure indexing works correctly.
+    </description>
+    <analytic>
+      This test minimizes the function x y cos(x+y), which when limited to 0..x..pi and 0..y..pi has a minimum at
+      (1.8218,1.8218) of -2.90946.
+    </analytic>
```

Comment:
Since it is analytic, do we have the related documentation?

Reply:
added